master-firmware/protobuf/position.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto2";
+
+import "nanopb.proto";
+
+message RobotPosition {
+    option (nanopb_msgopt).msgid = 10;
```

Comment:
This will conflict with the arm state message https://github.com/cvra/robot-software/blob/master/master-firmware/protobuf/manipulator.proto#L6
13 should be fine